contracts/StakingProxy.cdc

```diff
@@ -0,0 +1,121 @@
+// This contract defines an interface for node stakers
+// to use to be able to perform common staking actions
+
+// It also defines a resource that a node operator can
+// use to store staking proxies for all of their node operation
+// relationships 
+
+pub contract StakingProxy {
+
+    pub struct interface NodeDelegatorProxy {
```

Comment:
Minor, but maybe move this right below `NodeStakerProxy`. It's sort of a secondary interface and probably doesn't need to be at the top of the contract